_includes/operator/install.md

```diff
@@ -0,0 +1,71 @@
+## Install StorageOS operator
+
+The StorageOS cluster operator is a [Kubernetes native application](https://kubernetes.io/docs/concepts/extend-kubernetes/extend-cluster/) developed to
+deploy and configure StorageOS clusters, and assist with maintenance
+operations.
+
+The StorageOS operator is installed with Helm.
```

Comment:
`can be installed...`?
I think we will have a non-helm setup as well, right?

Reply:
Done, thank you.